includes/rest-api.php

```diff
@@ -819,12 +814,9 @@ public function pmpro_rest_api_recent_orders( $request ) {
 			
 			$results = $wpdb->get_results( $wpdb->prepare( $sql, $limit ) );
```

Comment:
Same as for line 755. Justification: You avoid the need to use the array index to address the correct object and the code becomes a little more readable.

Reply:
I think we'll revisit this area of the code in the future to utilize array_map on pmpro_format_date_iso8601 for addressing the column(s) needing to be formatted.